src/sentry/digests/utilities.py

```diff
@@ -78,16 +78,17 @@ def build_custom_digest(original_digest, events):
     return user_digest
 
 
-def build_events_by_actor(project_id, events, user_ids):
+def build_events_by_actor(
+    project_id: int, events: Iterable[Event], user_ids: Iterable[int]
+) -> Mapping[ActorTuple, Iterable[Event]]:
     """
-    build_events_by_actor(project_id: Int, events: Set(Events), user_ids: Set[Int]) -> Map[Actor, Set(Events)]
+    TODO(mgaeta): I know this is inefficient. ProjectOwnership.get_owners
```

Comment:
?

Reply:
Leaving a reminder for myself to address this in the follow-up PR.